integreat_cms/cms/migrations/0058_translations_management_settings.py

```diff
@@ -0,0 +1,86 @@
+# Generated by Django 3.2.16 on 2023-01-10 16:33
+
+from django.db import migrations, models
+from ..constants import roles
+
+
+# pylint: disable=unused-argument
+def update_roles(apps, schema_editor):
+    """
+    Update translation settings permissions
+
+    :param apps: The configuration of installed applications
+    :type apps: ~django.apps.registry.Apps
+
+    :param schema_editor: The database abstraction layer that creates actual SQL code
+    :type schema_editor: ~django.db.backends.base.schema.BaseDatabaseSchemaEditor
+    """
+    Group = apps.get_model("auth", "Group")
+    Permission = apps.get_model("auth", "Permission")
+
+    translations_management_permission = Permission.objects.get(
+        codename="manage_translations"
+    )
+
+    for role_name in dict(roles.STAFF_ROLES):
+        group = Group.objects.get(name=role_name)
+        group.permissions.add(translations_management_permission)
+
+    management_group = Group.objects.get(name="MANAGEMENT")
+    management_group.permissions.add(translations_management_permission)
```

Comment:
I have thought about this a lot - I think my current preference would be to just use the sledgehammer approach and unassign all permissions and re-assign them according to the config dict - this way, we avoid any mistakes that result from inconsistencies between those two definitions. 

```suggestion
    # Assign the correct permissions
    for role_name in dict(roles.CHOICES):
        group, _ = Group.objects.get_or_create(name=role_name)
        # Clear permissions
        group.permissions.clear()
        # Set permissions
        group.permissions.add(
            *Permission.objects.filter(codename__in=roles.PERMISSIONS[role_name])
        )
```

However, I'm still a bit unhappy with our current migration workflow, so we should probably find something more sensible in the long term (see #1857)